extensions/amp-fit-text/1.0/fit-text.js

```diff
@@ -59,7 +59,8 @@ export function FitText({
     if (!container || !content) {
       return;
     }
-    const observer = new ResizeObserver(() => resize());
+    const win = container.ownerDocument.defaultView;
```

Comment:
Use `toWin()`. Also, I found a flakiness with tests when I did https://github.com/ampproject/amphtml/pull/30955. Sometimes test completes before all effect have run. If the test's `realWin` gets released before the effect is run, then the effect will fail and it might have some confluence on other tests. Really hard to debug. The Preact `10.5` fixed that, but we can't upgrade yet due to some closure bugs. So for now, the best way is to do `if (!win) {return undefined}`. See [this change](https://github.com/ampproject/amphtml/pull/30955/files#diff-a7c674155aa449a6925a2f62ca43d6bb5eaca47ae9b78ed486a161df9183dde8R58-R62).

Reply:
I see, thanks for the pointer. Done.